cmd/detect.go

```diff
@@ -71,44 +71,48 @@ func runDetect(cmd *cobra.Command, args []string) {
 	// - git: scan the history of the repo
 	// - no-git: scan files by treating the repo as a plain directory
 	var (
-		findings []report.Finding
 		err      error
+		findings []report.Finding
 	)
 	if noGit {
-		paths, err := sources.DirectoryTargets(
-			source,
-			detector.Sema,
-			detector.FollowSymlinks,
-			detector.Config.Allowlists,
+		findings, err = detector.DetectSource(
+			&sources.Files{
+				Config:         &cfg,
+				FollowSymlinks: detector.FollowSymlinks,
+				MaxFileSize:    detector.MaxTargetMegaBytes * 1000000,
+				Path:           sourcePath,
+				Sema:           detector.Sema,
+			},
 		)
-		if err != nil {
-			logging.Fatal().Err(err).Send()
-		}
 
-		if findings, err = detector.DetectFiles(paths); err != nil {
-			// don't exit on error, just log it
-			logging.Error().Err(err).Msg("failed scan directory")
+		if err != nil {
+			logging.Fatal().Err(err)
```

Comment:
This is a no-op, right? The logger needs a terminal operator like `Msg()`, `Msgf()`, or `.Send()`.